lib/tasks/temp.rake

```diff
@@ -290,4 +290,9 @@ namespace :temp do
     end
   end
 
+  desc 'Look for a fix requests with linebreaks in titles'
+  task :remove_linebreaks_from_request_titles => :environment do
+    InfoRequest.where("title LIKE ?", "%\n%").each { |request| request.save! }
```

Comment:
It catches `\r\n` but not `\r`

`\r` on its own doesn't upset regex but it might be worth stripping out stray `\r`s (if there are any) as well for consistency